backend/resumatorController.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+/**
+ * resumatorController
+ * @module
+ * @param {Object} dbConf Database configuration
```

Comment:
object (lowercase) not Object. 
please add fields of object to ducmentation
http://stackoverflow.com/a/6460748